<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockEventDialog</class>
 <widget class="QDialog" name="BlockEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blokk műveletek</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="eventTable"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Törlés</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="writeBox">
       <property name="text">
        <string>Írás</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="readBox">
       <property name="text">
        <string>Olvasás</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="commLine"/>
     </item>
     <item>
      <widget class="QPushButton" name="commButton">
       <property name="text">
        <string>Megjegyzés</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
